File: Python/Pattern_d2.py
print("Enter the number of rows")
n=int(input())

print("Here is the pattern")

for i in range(1,n+1):
    j=i
    k=1
    while k<=i:
        print(j,end=" ")
        j-=1
        k+=1
    print("\n")

File: patternsJAVA/patternD2.java
/*
Pattern is:
1
2 1
3 2 1
4 3 2 1
*/
public class patternD2 {
    public static void main(String[] args) {
        int n = 4;
        for (int j = 1; j <= n; j++) {
            for (int i = j; i >= 1; i--) {
                System.out.print(i + " ");
            }
            System.out.println();
        }
    }
}
